### ISSUE TITLE
ONLY_PREVIEW_ALLOWED error

### ISSUE THREAD

**Issue by bcampeau** [alleyinteractive/apple-news#210]
We are seeing reports trickle in that users are hitting the ONLY_PREVIEW_ALLOWED error if Settings > Automatically publish to Apple News = Yes, similarly detailed here:

https://wordpress.org/support/topic/getting-a-preview-error-with-new-version

When a user is in the Post View, they encounter this error no matter how/when the “Preview” box is checked. Additionally, the meta box is not exposed until the user saves a draft or attempts to publish.

**Expected behavior**

- If Settings > Automatically publish to Apple News = Yes
- isPreview metadata is never sent (passive) unless specified by the user that the post is intended to be a Preview (isPreview=true)
- Note: A post cannot move back to isPreview=true after being published as isPreview=false; therefore the metadata isPreview=false should never be sent.
- In short, a hard flag of isPreview=false should never be sent to the Create Article endpoint.

**Additional Requests**

- Is it possible to always export the meta box as opposed to only exposing after first Draft/Publish?
- Can you remove the string “(All of these settings are required for publishing to Apple News.)” from the settings? Folks seem to be getting hung up on the Asynch setting: https://github.com/alleyinteractive/apple-news/issues/208
